modules/server_new/src/main/scala/observe/server/ODBSequencesLoader.scala

```diff
@@ -93,18 +93,17 @@ object ODBSequencesLoader {
   ): Sequence[F] = Sequence.sequence(id, atomId, toStepList(seq, overrides, d))
 
   private[server] def loadSequenceEndo[F[_]](
-    observer:   Option[Observer],
-    seqg:       SequenceGen[F],
-    execEngine: ExecEngineType[F],
-    l:          Lens[EngineState[F], Option[SequenceData[F]]]
+    observer: Option[Observer],
+    seqg:     SequenceGen[F],
+    l:        Lens[EngineState[F], Option[SequenceData[F]]]
   ): Endo[EngineState[F]] = st =>
     l.replace(
       SequenceData[F](
         observer,
         none,
         SystemOverrides.AllEnabled,
         seqg,
-        execEngine.load(
+        Engine.load(
```

Comment:
The method didn't use any of the instance state, so I moved it to the companion object. That removes the need to pass an instance of `Engine` around.